src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: bookReducer,
+});
```

Comment:
- [ ] Please import categories from src/redux/categories and add categories to `configureStore` method, see example below 👍🏿 

```suggestion

const store = configureStore({
  `reducer: { books: booksReducer, categories: categoriesReducer },`
});
```